daemon/daemon.go

```diff
@@ -537,12 +544,13 @@ func (d *Daemon) installIptablesRules() error {
 	// performed by kube-proxy for all packets destined for Cilium. Cilium
 	// installs a dedicated rule which does the source PAT to the right
 	// source IP.
+	clearMasqBit := fmt.Sprintf("%#04x/%#04x", 0, proxy.MagicMarkK8sMasq)
```

Comment:
The option to `--set-mark` is 32-bit, and is dumped as such in iptables output. So I'd replace `%#04x` with `%#08x` to be consistent with that format.

Same for all the others below.

Reply:
Added an extra commit to clean this up.